sample-lnd.conf

```diff
@@ -264,6 +264,11 @@
 ; for the channel closure is not set.
 ; coop-close-target-confs=10
 
+; The maximum time that is allowed to pass between receiving a channel state
+; update and signing the next commitment. Setting this to a longer duration
+; allows for more efficient channel operations at the cost of latency.
+; commit-time=50ms
```

Comment:
small nit: would you be opposed to something more like `channel-commit-interval`? 

two reasons in my head:
 - increase specificity and indicate that this relates to channels (we already have `db.batch-commit-interval`) and leave room for other types of commit paramters. could even be more specific, e.g. `channel-sign-commit-interval`
 - reuse the `interval` terminology already present in `db.batch-commit-interval` and in the much of the code base

Reply:
I picked `commit-time` because it matches the option name in `c-lightning`. Not a particular good reason. Renamed to what you're suggesting.